tests/helpers/test_framework_helpers.py

```diff
@@ -219,7 +237,11 @@ def test_find_suppliers_with_details_and_draft_service_counts(mock_data_client):
             ),
             'countersignedAt': '',
             'onFramework': True,
+            'signedAgreementReturnedAt': '2020-09-15T15:52:15.677327Z',
+            'signerName': 'gideon',
+            'signerRole': 'developer',
             'frameworkSlug': 'g-things-1',
+            'agreementId': ''
```

Comment:
I think the reason you are getting an empty string here is because you didn't set the `agreementId` on your mock data (possibly that was what `id` was supposed to be?)